build/lib/helm_require.sh

```diff
@@ -48,6 +48,8 @@ export IMAGE_TAG
 export HELM_REGISTRY
 envsubst <helm/sedfile.template >${SEDFILE}
 echo "s,version: v,version: ,g" >>${SEDFILE}
```

Comment:
This sed script is case sensitive, so this line above does change `version` and not `appVersion` although the `^` is better

Reply:
Yep, it works fine. I'm not changing that behavior, but I can add the `^` if you want me to.